app/assets/stylesheets/layout_components/buttons/button-grouping.scss

```diff
@@ -64,7 +64,6 @@
     width: 100%;
     display: flex;
     flex-direction: column;
-    justify-content: center;
```

Comment:
I believe I had added this when I was working on that card with Zoie to have the red button in automotive with the text and white button on top. I believe this will make it so it's not vertically centred. Did you happen to look at that?

Reply:
I don't think the automotive red box content will be vertically centered anymore, but there is still bottom padding on all of the elements (except the PTA, but that's cause it's missing an mb1 class), and equalizer so it still looks fine.
![image](https://user-images.githubusercontent.com/1906696/73766419-38331f00-4733-11ea-9852-9082d1288583.png)
 If we leave it in this is the issue that it's causing
![image](https://user-images.githubusercontent.com/1906696/73766194-ebe7df00-4732-11ea-8758-21cda591db5f.png)